.github/workflows/nightly.yml

```
@@ -2,7 +2,7 @@ name: Nightly checks
 
 on:
   schedule:
-    - cron: "0 0 * * *" # every day at midnight
+    - cron: "0 22 * * *" # daily at 10:00 PM
```

```suggestion
    - cron: "0 22 * * *" # daily at 10:00 PM (UTC)
```
Just adding the precision as for most of us it's actually not 10PM

---

makes sense 👍